<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Page01">
       <widget class="QLabel" name="loginLabel">
        <property name="geometry">
         <rect>
          <x>109</x>
          <y>6</y>
          <width>250</width>
          <height>250</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/RC/IMG1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="pwEdit">
        <property name="geometry">
         <rect>
          <x>129</x>
          <y>283</y>
          <width>207</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 10px;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string> Password</string>
        </property>
       </widget>
       <widget class="ChildLineEdit" name="otpEdit">
        <property name="geometry">
         <rect>
          <x>129</x>
          <y>321</y>
          <width>207</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 10px;</string>
        </property>
        <property name="placeholderText">
         <string> OTP Code</string>
        </property>
       </widget>
       <widget class="HoverButton" name="loginButton">
        <property name="geometry">
         <rect>
          <x>359</x>
          <y>280</y>
          <width>75</width>
          <height>71</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Log in</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="HoverButton" name="resetButton">
        <property name="geometry">
         <rect>
          <x>32</x>
          <y>280</y>
          <width>75</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QLabel" name="errorLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>259</y>
          <width>391</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>Incorrect Login. Try again.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Page02">
       <property name="styleSheet">
        <string notr="true">QTreeView::branch:has-siblings:!adjoins-item {
    border-image: url(vline.png) 0;
}

QTreeView::branch:has-siblings:adjoins-item {
    border-image: url(:/RC/branch-more.png) 0;
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(:/RC/branch-end.png) 0;
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:/RC/branch-closed.png);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
        border-image: none;
        image: url(:/RC/branch-open.png);
}
</string>
       </property>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>2</y>
          <width>221</width>
          <height>358</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border: 2px solid white;
border-radius: 5px;
QTreeView {
        QScrollBar:vertical {
            border: 2px solid grey;
            background: #32CC99;
            height: 15px;
            margin: 0px 20px 0 20px;
        }
    }</string>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="HoverButton" name="urlButton">
        <property name="geometry">
         <rect>
          <x>226</x>
          <y>2</y>
          <width>241</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
       <widget class="HoverButton" name="usernameButton">
        <property name="geometry">
         <rect>
          <x>227</x>
          <y>44</y>
          <width>241</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
       <widget class="HoverButton" name="pwButton">
        <property name="geometry">
         <rect>
          <x>227</x>
          <y>87</y>
          <width>241</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>228</x>
          <y>131</y>
          <width>241</width>
          <height>229</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 5px;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChildLineEdit</class>
   <extends>QLineEdit</extends>
   <header>childlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>HoverButton</class>
   <extends>QPushButton</extends>
   <header>hoverbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
